### PR TITLE
fixed: content registry changed to lowercas; added known links

### PR DESCRIPTION
**Pull request by AlexanderBazukevich**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,66 +6,64 @@
...
+                <h1 class="article__title">fustce faucibusy ante libero</h1>
                 <p class="article__content">Cras interdum vestum a dolor amet tursisi ligula rhoncus sodales...</p>
             </div>
         </div>
         
         <div class="content-section">
-            <h1 class="content-section__title"></h1>
+            <h1 class="content-section__title">the gallery</h1>
             <div class="content-section__border"></div>
             <div class="gallery">
```
Comment (denisnarush) [inline index.html:68]: I think that we can easy use one `gallery` block, with 4 elements inside. Can we?

File: index.html [hunk 2]
```diff
@@ -77,14 +75,12 @@ <h1 class="content-section__title"></h1>
             </div>
 
             <div class="cover">
-                <h1 class="title title_center title_small" >PASSION <B>FREE</B> RESPONSIVE EMAIL TEMPLATE</h1>
-                <a class="link link_button_white link_text_small"></a>
+                <h1 class="title title_center title_small" >passion <span class="title__part title__part_bold">free</span> responsitive email template</h1>
```
Comment (denisnarush) [inline index.html]: unnecessary whitespace `" >pass`.

**Review by denisnarush — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (AlexanderBazukevich) [reply index.html:68]: Should I create modificators for "gallery__link" elements to split gallery in two lines?

Comment (denisnarush) [reply index.html:68]: No

**Review by denisnarush — CHANGES_REQUESTED** [review]

**Review by denisnarush — APPROVED** [review]
